nats.go

```diff
@@ -480,8 +480,10 @@ const (
 	// NUID size
 	nuidSize = 22
 
-	// Default port used if none is specified in given URL(s)
-	defaultPortString = "4222"
+	// Default ports used if none is specified in given URL(s)
```

Comment:
Sorry, these changes are part of the next PR, will revert changes to nats.go and force push...